<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="num_pushButton_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enter_pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_0">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="q_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
